web-client/src/views/Messages/MessageTable.tsx

```diff
@@ -142,19 +142,26 @@ export const MessageTable = connect<
             </div>
           </div>
         )}
-        <div className="overflow-x-auto" id={id}>
```

Comment:
This was wrapped around the wrong section and made the horizontal scrollbar appear spuriously at time